README.markdown

```diff
@@ -9025,6 +9026,17 @@ This API was first introduced in the `0.9.5` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+ngx.worker.pids
+--------------
+
+**syntax:** *pids = ngx.worker.pids()*
+
+**context:** *set_by_lua&#42;, rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, header_filter_by_lua&#42;, body_filter_by_lua&#42;, log_by_lua&#42;, ngx.timer.&#42;, init_by_lua&#42;, init_worker_by_lua&#42;, exit_worker_by_lua&#42;*
+
+This function returns a Lua table for all Nginx worker process ID (PID). Nginx uses channel to send the current worker PID to another worker in the worker process start or restart. So this API can get all worker PID at any time. `init_by_lua` and `init_worker_by_lua` maybe can not get all worker PID because in these phases other worker isn't started.
```

Comment:
can we just disable the API in init_by_lua and init_worker_by_lua

Reply:
done